go-controller/pkg/cluster/management-port.go

```diff
@@ -10,11 +10,11 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func createManagementPortGeneric(nodeName, localSubnet string, clusterSubnet []string) (string, string, string, string, error) {
+func createManagementPortGeneric(nodeName, localSubnet string, clusterSubnet []string) (string, string, string, string, string, error) {
 	// Determine the IP of the node switch's logical router port on the cluster router
```

Comment:
The function argument `clusterSubnet` is not being used inside the function, so you can remove it.